app/lib/bk/compat/models/steps/command_step_extensions.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module BK
+  module Compat
+    # Command step extensions
+    class CommandStep < BaseStep
+      def stack=(stack_value)
```

Comment:
commands should not have a `stack` element because they are not part of a buildkite step